src/components/code/_code_block.tsx

```diff
@@ -55,6 +55,10 @@ interface Props {
   overflowHeight?: number;
   paddingSize: PaddingSize;
   transparentBackground: boolean;
+  /**
+   * specify how white-space inside the element is handled
+   */
+  whiteSpace?: 'nowrap' | 'normal' | 'pre';
```

Comment:
As suggested by @cchaos, the options should be:

```suggestion
  whiteSpace?: 'pre' | 'pre-wrap';
```

Also, you should explain better what is `pre` and `pre-wrap` doing. And you can also mention that `pre-wrap` is the `default` prop.